src/components/Person/Person.jsx

```diff
@@ -1 +1,42 @@
-// export const Person = ({ person }) => ();
+import React from "react";
+
+export const Person = ({ person }) => {
+    const { 
+        name,
+        age,
+        sex,
+        isMarried = false,
```

Comment:
It's not necessary, bc if you will be have undefined => Boolean(undefined) === false
```suggestion
        isMarried,
```